nntrainer/layers/cl_layers/concat_cl.h

```diff
@@ -148,15 +149,13 @@ class ConcatLayerCl : public Layer {
    * @param[in] input1_height   represents the height of the input tensor
    * @param[in] input1_width   represents the width of the input tensor A
    * @param[in] input2_width   represents the width of the input tensor X
-   * @param[in] context RunLayerContext reference
    */
   void concat_cl_axis3_fp16(const __fp16 *matAdata, const __fp16 *vecXdata,
```

Comment:
Please add `ENABLE_FP16` condition to this header.
```suggestion
#ifdef ENABLE_FP16
  void concat_cl_axis3_fp16(const __fp16 *matAdata, const __fp16 *vecXdata,
```
Besides this function prototypes, all concat_cl_axis*_fp16 should be imported only when ENABLE_FP16 is defined.
Please consider this conditional compilation for cpp file as well.